tutorial/lesson-3/README.md

```diff
@@ -223,6 +235,8 @@ Then add an `add_update` method to the flow class. It is just a copy of the
 `WAB::UI::RestFlow#add_update` method with the transition modified and the new
 `EntryUpdate` class use instead of the `WAB::UI::Update` class.
 
+`lib/entry_flow.rb`
```

Comment:
Adding a tag for which file is in , is a great addition.
It also makes it much faster to go through if you already did the tutorial and just need some reference.
No need to read the description above and understand where to copy it to.

Small comment with a personal preference , I saw a lot of places putting the file name as a comment in the first line of the file content.
Might be easier to maintain in the long run since it's in the same block as the content.